ui/src/widgets/ui-dropdown/UIDropdown.vue

```diff
@@ -94,14 +104,19 @@ export default {
                 // 2. update the selected value(s)
                 this.select(payload)
             }
-            // additionally, we need to support both single and multi selection
-            // For now, we only support selecting which item(s) are selected, not updating the available options
-            // if the payload is an array, we assume it is a list of values to select
+
+            if (msg.label !== undefined) {
+                this.dynamic.label = msg.label
+            }
+
+            if (msg.multiple !== undefined) {
+                this.dynamic.multiple = msg.multiple
+            }
```

Comment:
```suggestion
            if (typeof msg.label !== 'undefined') {
                this.dynamic.label = msg.label
            }

            if (typeof msg.multiple !== 'undefined') {
                this.dynamic.multiple = msg.multiple
            }
```

`typeof x !== 'undefined'` is a more robust and safe way to check for the existence of a variable